src/loginPage/LoginPage.js

```diff
@@ -1,15 +1,65 @@
-import React from 'react';
+import React, {useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+
+//TODO: replace to a separate file
+async function newSession(login, password){
+    //TODO: replace to constants
+    const SERVER_PATH = 'http://localhost:5000';
+    const endPoint = SERVER_PATH + '/sessions/login';
+
+    return await (fetch(
+        endPoint,
+        { method: 'post',
+            body: JSON.stringify({ login, password }),
+            headers: { 'Content-type': 'application/json'}
+        })
+        .then(res => {
+            if (res.ok)
+                return res.json();
+            throw res;
+        }).then(data => {
+            return { status: data.status, sessionID: data.sessionID, userRole: data.role};
+        })
+        .catch(err => {
+            return {status: err.status, massage: err.statusText};
+        }));
+}
 
 export function LoginPage() {
+    const navigate = useNavigate();
+    const [login, setLogin] = useState('');
+    const [password, setPassword] = useState('');
+    const [authErrorText, setAuthErrorText] = useState('');
+
+    const handleLoginButtonClick = async () => {
+        const result = await newSession(login, password);
+
+        if (result.status === 401){
+            return setAuthErrorText('Login or password is incorrect');
+        }else if (result.status === 403){
+            return setAuthErrorText('User is blocked');
+        }
+
+        setAuthErrorText('');
+
+        // TODO: get rid of magic values
+        if(result.userRole === 1){
+            navigate('/administrator-panel/');
+        }else if(result.userRole === 2){
```

Comment:
I'd suggest you to make eslint and prettier work together :)
Spaces between `} else if (` and other places would make your code more readable

Reply:
Ok, I will do it in next PR